docs/_include_files/options_def.rst

```diff
@@ -373,7 +373,8 @@ The directory path for root of generated files. A directory is created if
 it does not exist. Default: ``dpct_output``.
 
 The relative paths for the generated files are maintained, and the
-extension is changed as follows:
+extension is changed as follows by default, the behavior can be changed
```

Comment:
Recommend the following edit: 

"The relative paths for the generated files are maintained. By default, file extensions are changed as follows:

*.cu → *.dp.cpp
....
*.h *.hpp *.hxx → extensions are kept the same


To limit file extension changes to ``.cu`` and ``.cuh`` files only, use the ``--change-cuda-files-extension-only`` option."

Reply:
fixed